### PR TITLE
Alias for loopback interface

### PR DESCRIPTION
**Pull request by nickvarepam**
Adding alias(lo:1) for loopback interface with 10.0.0.7/24 address, via adding localalias.network file to systemd/network folder.

### CODE REVIEW

File: recipes-extended/netconfig/files/localalias.network [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Match]
+Name=lo
+
+[Network]
+
+[Address]
+Label=lo:1
+Address=10.0.0.7/24
+[Route]
```
Comment (lorc) [inline recipes-extended/netconfig/files/localalias.network:9]: Missing the EOL at EOF.

**Review by lorc — CHANGES_REQUESTED** [review]
What is the purpose of this change? Could you please elaborate your commit message?

File: recipes-core/images/aos-image-minimal.inc [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 BOARD_MODEL ?= "vm-dev;1.0"
 
-BUNDLE_IMAGE_VERSION ?= "3.1.0"
+BUNDLE_IMAGE_VERSION ?= "3.1.1"
```
Comment (Eabramov84) [inline recipes-core/images/aos-image-minimal.inc:5]: not related changes. Version control will be defined later.

File: recipes-extended/netconfig/files/localalias.network (hunk 1 above)
Comment (nickvarepam) [reply recipes-extended/netconfig/files/localalias.network]: Commit message is updated.
EOL at EOF is added.

File: doc/aos_board.cfg [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  ],
+  "components": [
+    {
+      "id": "rootfs"
+    },
+    {
+      "id": "boot"
+    }
+  ],
+  "vendorVersion": "9",
```
Comment (Eabramov84) [inline doc/aos_board.cfg:21]: remove vendorVersion and vendorVersion fields.
Add board configuration file as separate commit

File: recipes-extended/netconfig/files/localalias.network [hunk 4]
```diff
@@ -0,0 +1,9 @@
+[Match]
```
Comment (Eabramov84) [inline recipes-extended/netconfig/files/localalias.network:1]: rename to lo.1.network

**Review by Eabramov84 — COMMENTED** [review]
LGTM

File: recipes-extended/netconfig/files/lo.1.network [hunk 5]
```diff
@@ -0,0 +1,9 @@
+[Match]
+Name=lo
+
+[Network]
```
Comment (al1img) [inline recipes-extended/netconfig/files/lo.1.network:4]: Why do you need empty [Network] and [Route] nodes?

File: recipes-extended/netconfig/files/localalias.network (hunk 4 above)
Comment (nickvarepam) [reply recipes-extended/netconfig/files/localalias.network]: done

File: doc/aos_board.cfg (hunk 3 above)
Comment (nickvarepam) [reply doc/aos_board.cfg]: done

File: recipes-extended/netconfig/files/lo.1.network (hunk 5 above)
Comment (nickvarepam) [reply recipes-extended/netconfig/files/lo.1.network]: Sounds fair, nodes are removed.

File: doc/aos_board.cfg [hunk 6]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (al1img) [inline doc/aos_board.cfg:1]: Could you use tab width 4?

**Review by al1img — APPROVED** [review]

**Pull request merged**